core/metacling/src/CMakeLists.txt

```diff
@@ -74,11 +74,112 @@ if(MSVC)
    if(CMAKE_SIZEOF_VOID_P EQUAL 8)
      set_source_files_properties(TCling.cxx COMPILE_FLAGS $<$<CONFIG:Debug>:/bigobj>)
    endif()
+ endif()
+
+##### libCppInterOp ############################################################
+include(ExternalProject)
+
+# CppInterOp Libraries
+set(_cppinterop_build_type ${CMAKE_CFG_INTDIR})
+if(MSVC)
+  if (winrtdebug)
+    set(_cppinterop_build_type Debug)
+  else()
+    set(_cppinterop_build_type Release)
+  endif()
+endif(MSVC)
+if(NOT _cppinterop_build_type STREQUAL "" AND NOT _cppinterop_build_type STREQUAL ".")
+  set(EXTRA_BUILD_ARGS --config ${_cppinterop_build_type})
+endif()
+set(_CPPINTEROP_BUILD_PATH ${CMAKE_CURRENT_BINARY_DIR}/cppinterop-prefix/src/cppinterop-build/${_cppinterop_build_type}/)
+set(_CPPINTEROP_LIBRARY_PATH ${_CPPINTEROP_BUILD_PATH}/lib${LLVM_LIBDIR_SUFFIX})
+
+# build byproducts only needed by Ninja
+if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
+  set(CPPINTEROP_BYPRODUCTS
+     ${_CPPINTEROP_LIBRARY_PATH}/${CMAKE_STATIC_LIBRARY_PREFIX}clangCppInterOp${CMAKE_STATIC_LIBRARY_SUFFIX}
+  )
+endif()
+
+if(APPLE)
+  set(_cppinterop_extra_cmake_args -DCMAKE_OSX_SYSROOT=${CMAKE_OSX_SYSROOT})
 endif()
 
-add_dependencies(MetaCling CLING)
+if (CMAKE_CXX_STANDARD)
+  list(APPEND _cppinterop_extra_cmake_args -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD})
+endif(CMAKE_CXX_STANDARD)
+
+if (LLVM_FORCE_USE_OLD_TOOLCHAIN)
```

Comment:
This is defined in `interpreter/CMakeLists.txt`, so very likely we cannot access it here...

Reply:
We shouldn't need this as we build InterOp with clang>6 and hence gcc>=7.1